testsuite/MDAnalysisTests/analysis/test_rms.py

```diff
@@ -128,6 +129,70 @@ def test_with_superposition_equal(self):
         rmsd_superposition = rms.rmsd(A, B, center=True, superposition=True)
         assert_almost_equal(rmsd, rmsd_superposition)
 
+class TestRMSD(object):
+    def setUp(self):
+        self.universe = MDAnalysis.Universe(PSF, DCD)
+        self.tempdir = tempdir.TempDir()
+        self.outfile = os.path.join(self.tempdir.name, 'rmsd.npy')
+
+    def tearDown(self):
+        del self.universe
+        del self.tempdir
+
+    def test_rmsd(self):
+        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA')
+        RMSD.run(step=49)
+        correct_values = [[  0, 0, 0],
+                          [  49, 48.9999, 4.68953]]
+        assert_almost_equal(RMSD.rmsd, correct_values, 4,
+                            err_msg="error: rmsd profile should match test " +
+                            "values")
+
+
+    def test_rmsd_single_frame(self):
+        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA')
+        RMSD.run(start=5, stop=6)
+
+
+    def test_mass_weighted_and_save(self):
+        RMSD = MDAnalysis.analysis.rms.RMSD(self.universe, select='name CA')
+        RMSD.run(step=49, mass_weighted= True)
+        RMSD.save()
```

Comment:
read the save file and compare to the `correct_values` (see above) to `assert` correct file writing.